test/test_filters.py

```diff
@@ -35,3 +36,22 @@ def test_get_element_by_tag_nested():
     """, 'div')
     print(result)
     eq_(result, """<div>foo</div><div>bar</div>""")
+
+
+def test_json_format_filter():
+    json_format_filter = JsonFormatFilter(None, None)
+    result = json_format_filter.filter(
+        """{"field1": {"f1.1": "value"},"field2": "value"}""")
+    print(result)
```

Comment:
No need for a print here.

Reply:
I put this print here because all existing filter tests already have print in the same place. When test fails there isn't any way to see what the result actually contains unless it's printed here. And when tests are run with nosetests it only outputs these prints when tests fail. So I do think it makes sense to keep this print here.